recipes/mod_alias.rb

```diff
@@ -18,5 +18,5 @@
 #
 
 apache_module 'alias' do
-  conf true
+  conf false
```

Comment:
This specifies that a configure exists and should be installed. If you remove this a config will never be deployed and would need to be deployed manually breaking parity with ASF defaults.